projects/ngx-sub-form/src/lib/ngx-sub-form.component.spec.ts

```diff
@@ -605,22 +606,23 @@ class SubArrayComponent extends NgxSubFormRemapComponent<Vehicle[], VehiclesArra
     return formValue.vehicles;
   }
 
-  public createFormArrayControl(key: ArrayPropertyOf<VehiclesArrayForm> | undefined): FormControl {
-    return new FormControl(null, [Validators.required]);
+  public createFormArrayControl(
+    key: ArrayPropertyOf<VehiclesArrayForm> | undefined,
+    initialValue: ArrayTypeOfPropertyOf<VehiclesArrayForm>,
```

Comment:
`initialValue: ArrayTypeOfPropertyOf<VehiclesArrayForm>`

should be 

`initialValue: ArrayTypeOfPropertyOf<VehiclesArrayForm> | null`